### ISSUE TITLE
Parser requires meta object

### ISSUE THREAD

**Issue by Logicer16** [yeonjuan/html-eslint#184]
When attempting to use `@html-eslint/parser`, `eslint --cache` throws an error as the parser lacks a [meta object](https://eslint.org/docs/latest/extend/custom-parsers#meta-data-in-custom-parsers):

```
Error: Could not serialize parser object (missing 'meta' object).
```

Config (eslint.config.js):
```js
import htmlPlugin from "@html-eslint/eslint-plugin";
export default [
  htmlPlugin.configs["flat/recommended"]
];
```

**Comment by yeonjuan** [comment]
Thanks for the report, it looks like we need to change the parser to export the meta object.

**Comment by Logicer16** [comment]
Did you mean to tag this as a bug instead of an enhancement? This is a fatal error being thrown. The parser does not work in a flat config. I'm surprised no one noticed this earlier.

**Comment by yeonjuan** [comment]
Hi @Logicer16. I tried flat config and couldn't see any errors, could you please share your eslint version and the settings you used?

> Did you mean to tag this as a bug instead of an enhancement? This is a fatal error being thrown.

Right, I'll change the label. Thanks!

**Comment by Logicer16** [comment]
> I tried flat config and couldn't see any errors

I've taken a look, it only seems to be an issue when the `--cache` option is used. That's probably why it wasn't noticed earlier.

**Comment by yeonjuan** [comment]
@Logicer16 

> I've taken a look, it only seems to be an issue when either --cache or --print-config is used. That's probably why it wasn't noticed earlier.

I've confirmed that the error you mentioned is reproduced! Thanks for the details. I'll fix it up and release it this weekend, but if you're planning to contribute yourself, let me know. I look forward to it.

**Comment by yeonjuan** [comment]
Hi @Logicer16 , It's fixed and released in `@html-eslint/parser@0.24.1` , `@html-eslint/eslint-plugin@0.24.1`. thanks!

**Comment by Logicer16** [comment]
Thank you so much! ❤️
